aspnetcore/web-api/jsonpatch.md

```diff
@@ -218,7 +218,7 @@ This operation is functionally the same as a `move` operation without the final
 The following sample patch document:
 
 * Copies the value of `Orders[0].OrderName` to `CustomerName`.
-* Inserts a copy of `Orders[1]` before `Orders[0]`.
```

Comment:
here, i guess you should change description for patch or change patch.

Reply:
While this is correct, I think a better change would be to change the sample to do this:

```
{
    "op": "copy",
    "from": "/orders/1",
    "path": "/orders/0"
  }
```

It keeps it similar to the Move operation and would make this comment correct

Reply:
totally agree.

Reply:
have created one other PR.